src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
```

Comment:
The padding for the left and right sides of the card is 16px, which is fine, but the top padding is 32px. This may lead to the card content not being vertically centered. Please, check this.